WORKSPACE

```diff
@@ -32,8 +32,15 @@ new_go_repository(
 
 git_repository(
     name = "io_bazel_rules_docker",
+    commit = "db1b348dfdf161a784bc1efc5a1020395572b996",
     remote = "https://github.com/bazelbuild/rules_docker.git",
-    tag = "v0.0.1",
+
+)
+
+git_repository(
+    name = "containerregistry",
+    commit = "b0278a1544238d03648861b6d9395414d4c958e5",
+    remote = "https://github.com/google/containerregistry",
 )
 
 load(
```

Comment:
Could you try pulling mock into the WORKSPACE via something like this:
https://github.com/bazelbuild/rules_k8s/pull/5/files#diff-09498dbadf45966909850dc8a47ebb13R46